### ISSUE TITLE
[Bug] Tooltip with renderMode: 'richText' does not apply rich styling to formatter.

### ISSUE THREAD

**Issue by GidianB** [apache/echarts#19966]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=bar-simple&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurGAJ4gCmRA5AMYCGYVA5sAE7m0A0J6AE2aMiAbVoBZGL1i0AKgFcqM2gHUqAlXIAWClQDEOEFQGVmphdFoBdEgF8-6cjnxFUpMpRqyAbowA2SrT2jmTAwP6QIG78sBxU0AJUHFJJdEb02nJUGGC8sQBmnAC2zCwcRAAUIIwcjMV4AJTo8AB8xB7o8WAKHHCwAAawACRINXUNAHTQ9VR2bgBGgVQAPqPj9XiTfst2dgOxDrEA9McAQlT-wADusHgU_hDQrLD4sNDAYLCMICCPGrEMtoYp1YEslCDQbB6BFOHRwcpQqCitAwOoIKxtGB4RFNEjSHZDviWLkTA9vO5OkDIaCETSoTCrhVZAj8lDSCi0VQMVicf48bEPITQcKCSESHhkhAqIRYKJYpSPEIwCI5QBGABMAAYeLBtTrYGqAKwGgAcBoA7Aa1WrrQBmLXWfHoCjUeG1YJCki2OwAbiAA

### Steps to Reproduce

Specify renderMode: 'richText' in the tooltip and use richText formatting in the formatter.
Then specify the richText styling in tooltip.rich or in tooltip.textStyle.rich

### Current Behavior

Currently, rich is not supported by tooltip, and renderMode:'richText' seems to be a useless option. In wechat applets without DOM, there is no way to customize different styles.

### Expected Behavior

rich styling should be applied to the formatter.

### Environment

```markdown
- OS: Windows 10
- Browser: Chrome
```

### What does the proposed API look like?
`tooltip.rich`
or
`tooltip.textStyle.rich`

**Comment by helgasoft** [comment]
Agree, renderMode 'richText' is misleading since there is no _rich_ in tooltip API.
Technically we cannot call it a bug since _rich_ is missing, but I think _rich_ was **meant to be** used, just some source code was not hooked up...?
Proof: tooltip **recognizes** 'rred' as rich format and does not display it as string, but does not apply it as format either.
```
  formatter: '{b}: {rred|{c} }',
  renderMode: 'richText',
  rich: {
    primary: { color: 'magenta' },
    rred: { color: 'red', fontWeight: 'bold' }
  },
```
So this issue could be updated to a valid and very useful New Feature Request.

**Comment by Ovilia** [comment]
@helgasoft Thanks for the information! I will review the PR tomorrow.

**Comment by GidianB** [comment]
@Ovilia when will this be resolved?

**Comment by gufengpiaoyi** [comment]
any update on this?

**Comment by zhanzz** [comment]
any update on this?

**Comment by Travelguest** [comment]
any update on this?

**Comment by tubloko** [comment]
any update on this?

**Comment by XHL-study** [comment]
+1

**Comment by XHL-study** [comment]
暂时性的解决方案
https://docs.qq.com/doc/p/4b05780aefa6438832994bba7a6a3c21a4456515

**Comment by GetemJay** [comment]
6.0都出了 这个问题还没解决吗亲？
